WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+extension Bundle {
+    static let app: Bundle = {
+        var url = Bundle.main.bundleURL
+        while url.pathExtension != "app" && url.lastPathComponent != "/" {
+            url.deleteLastPathComponent()
+        }
+        guard let appBundle = Bundle(url: url) else { fatalError("Unable to find the parent app bundle") }
+        return appBundle
+    }()
+}
+
+/// This is mostly useful to express *intent* on your API. It does not provide any compile-time guarantee
```

Comment:
Given this is a documentation comment, how do you feel about making it more documentation-y?

```suggestion
/// A `typealias` of `String`. Use this to express *intent* on your API, but remember it won't not provide any compile-time guarantee.
```

Reply:
I like the imperative wording of "Use this to…". I feel like "A typealias of String" part though is just like saying `int x = 1 /* assign 1 to x */` aka a self-explanatory, redundant comment?

What about this then?
```suggestion
/// Use this to express *intent* on your API – but remember that, as a `typealias`, it won't provide any compile-time guarantee.
```